### ISSUE TITLE
Proof of ID

### ISSUE THREAD

**Issue by lucyTKS** [SmoofCreative/manor-house#21]
Location: Checkout

User: Admin

Issue: As the admin, I need to gather proof of ID from customers before signing off on an order. I must be able to collect either UK driving licence, UK passport or international passport number.

Notes: 
- This option is optional
-  There is a flow chart with amounts, if the amount is higher than shown in the flow chart, we need that number. 
- No need for V1. 
- Would also need a process in place to ensure customer is not doing many small transfers to stay under the limit. (based on Ip address of cookie.) 
- Icebox. 

@tomstarley says... Looks interesting! This task is going to need some thought.  We need some way to validate form the start. And there's options:

1. Manor FX could call the user after the transfer has been made for home deliveries. And we could ask click and collect customers to bring in ID when they collect the order. This is the easiest and quickest approach to dev. It is also the approach which the least friction for the customer (at checkout). BUT it takes man power and leaves room for mistakes.

2. We could use a form field to enter in the respective ID number which can then either be automatically or manually checked (like Covent garden FX). Still room for error here as the user could enter it incorrectly and possibly this is still a manual process meaning some of this would have to be automated or checked anyway. This approach will also frustrate the user as they will not want to enter in the number. So currently option 1 seems better.

3. Option 3 use a fully automated photo service like you have suggested. No margin for error, no people power and relatively easy to do for the user (still not perfect). But could take some time to integrate (@dale), and may cost per use.

Personally i think we go with option 1, then move to option 3 at later date. @Mario Van Poppel what are your thoughts?

**Comment by tomstarley** [comment]
@LeftoverCurrency need your final say on this one too? :)

**Comment by LeftoverCurrency** [comment]
@tomstarley having given this consideration. I agree with you that we need to have the least amount of friction at checkout as possible. We want to have the user put in the order (buy or sell) without any friction. 

The user can then indeed choose to make payment right away or pay at collection. For selling currency the user can choose between sending the currency to us by post or self-delivering it. 

Once we have received the payment, or the currency, we will then contact the user to obtain the required ID documents and extra pieces of information (if any). 

The types of ID and extra information we need (e.g. source of funds) varies per transaction. They are based on the amount, the type of customer and even the type of currency involved (high risk countries vs low risk countries). This is too complex for now. And we want to keep it simple for everyone, including you guys. Therefore we will indeed go for option 1. 

The only thing we have to make sure of is that during checkout the user's telephone number is obligatory.

**Comment by LeftoverCurrency** [comment]
Added to the above: If the user decides to collect / self-deliver, we need to know this in advance. That way we can advise what ID / information we need from the customer. So that when they come to the shop they don't get frustrated that we only tell them then.

**Comment by LeftoverCurrency** [comment]
Hope this all makes sense. Back to you now.

**Comment by tomstarley** [comment]
Yup sounds good! So thinking it through you can either contact them before they make the transfer but after the order or contact them after the transfer has cleared. 

The issue will be if they want to click and collect the next day and a staff member forgets to call them 
 or can't get hold of them. There will be a small window of opportunity that Fernanda will need to be aware off. We can automate this in the future. I shall create a new task for that.
